custom_components/tuya_local/devices/monzana_klimaanlage_air_conditioner.yaml

```diff
@@ -0,0 +1,78 @@
+name: Air conditioner
+products:
+  - id: 000004y1yr
```

Comment:
This doesn't seem quite like the format a product_id normally takes (though it is theoretically possible). Is it taken from the iot.tuya.com API Explorer, or some other model id from somewhere else?

Reply:
I think it was from api explorer but I'll double check in the evening.

Reply:
Hmm, so I again went to API explorer and it returns:
```
{
  "result": {
    "model": "{\"modelId\":\"000004y1yr\",\"services\":[{\"actions\":[],\"code\":\"\",\"description\":\"\",\"events\":[],\"name\":\"默认服务\",\"properties\":[{\"abilityId\":1,\"accessMode\":\"rw\",\"code\":\"switch\",\"description\":\"\",\"extensions\":{\"attribute\":\"129\"},\"name\":\"开关\",\"typeSpec\":{\"type\":\"bool\",\"typeDefaultValue\":false}},{\"abilityId\":2,\"accessMode\":\"rw\",\"code\":\"temp_set\",\"description\":\"\",\"extensions\":{\"attribute\":\"128\"},\"name\":\"温度设置\",\"typeSpec\":{\"max\":32,\"min\":16,\"scale\":0,\"step\":1,\"type\":\"value\",\"typeDefaultValue\":16,\"unit\":\"℃\"}},{\"abilityId\":3,\"accessMode\":\"ro\",\"code\":\"temp_current\",\"description\":\"\",\"extensions\":{\"attribute\":\"128\"},\"name\":\"当前温度\",\"typeSpec\":{\"max\":99,\"min\":0,\"scale\":0,\"step\":1,\"type\":\"value\",\"typeDefaultValue\":0,\"unit\":\"℃\"}},{\"abilityId\":4,\"accessMode\":\"rw\",\"code\":\"mode\",\"description\":\"\",\"extensions\":{\"iconName\":\"icon-dp_mode\",\"attribute\":\"128\"},\"name\":\"工作模式\",\"typeSpec\":{\"range\":[\"Cool\",\"Dyr\",\"Fan\",\"Heat\"],\"type\":\"enum\",\"typeDefaultValue\":\"Cool\"}},{\"abilityId\":5,\"accessMode\":\"rw\",\"code\":\"fan_speed_enum\",\"description\":\"\",\"extensions\":{\"iconName\":\"icon-dp_wind\",\"attribute\":\"128\"},\"name\":\"风速\",\"typeSpec\":{\"range\":[\"Low\",\"High\"],\"type\":\"enum\",\"typeDefaultValue\":\"Low\"}},{\"abilityId\":14,\"accessMode\":\"rw\",\"code\":\"child_lock\",\"description\":\"\",\"extensions\":{\"iconName\":\"icon-tcl_function_hs\",\"attribute\":\"128\"},\"name\":\"童锁\",\"typeSpec\":{\"type\":\"bool\",\"typeDefaultValue\":false}},{\"abilityId\":19,\"accessMode\":\"rw\",\"code\":\"temp_unit_convert\",\"description\":\"\",\"extensions\":{\"attribute\":\"128\"},\"name\":\"温标切换\",\"typeSpec\":{\"range\":[\"c\",\"f\"],\"type\":\"enum\",\"typeDefaultValue\":\"c\"}},{\"abilityId\":22,\"accessMode\":\"ro\",\"code\":\"fault\",\"description\":\"\",\"extensions\":{\"scope\":\"fault\",\"attribute\":\"128\"},\"name\":\"故障告警\",\"typeSpec\":{\"label\":[\"Rtemp_fault\",\"IPtemp_fault\",\"Water_FULL\"],\"maxlen\":3,\"type\":\"bitmap\",\"typeDefaultValue\":0}},{\"abilityId\":23,\"accessMode\":\"rw\",\"code\":\"temp_set_f\",\"description\":\"\",\"extensions\":{\"attribute\":\"128\"},\"name\":\"温度设置-℉\",\"typeSpec\":{\"max\":90,\"min\":61,\"scale\":0,\"step\":1,\"type\":\"value\",\"typeDefaultValue\":61,\"unit\":\"℉\"}},{\"abilityId\":24,\"accessMode\":\"ro\",\"code\":\"temp_current_f\",\"description\":\"\",\"extensions\":{\"attribute\":\"128\"},\"name\":\"当前温度 ℉\",\"typeSpec\":{\"max\":99,\"min\":0,\"scale\":0,\"step\":1,\"type\":\"value\",\"typeDefaultValue\":0,\"unit\":\"℉\"}},{\"abilityId\":101,\"accessMode\":\"rw\",\"code\":\"Sleep\",\"description\":\"\",\"extensions\":{\"iconName\":\"icon-a_function_sleep\"},\"name\":\"睡眠\",\"typeSpec\":{\"type\":\"bool\",\"typeDefaultValue\":false}},{\"abilityId\":102,\"accessMode\":\"ro\",\"code\":\"Client_ID\",\"description\":\"\",\"name\":\"客户型号\",\"typeSpec\":{\"max\":99,\"min\":0,\"scale\":0,\"step\":1,\"type\":\"value\",\"typeDefaultValue\":0,\"unit\":\"\"}},{\"abilityId\":103,\"accessMode\":\"rw\",\"code\":\"Model_ID\",\"description\":\"C: 单冷\\nC_H: 冷暖\",\"name\":\"机型上报\",\"typeSpec\":{\"range\":[\"C\",\"C_H\"],\"type\":\"enum\",\"typeDefaultValue\":\"C\"}}]}]}"
  },
  "success": true,
  "t": 1688070756711,
  "tid": "158ac67716bc11ee97740a4306e7517e"
}
```
That string I used was `modelId`. Or should it be `tid`?

Reply:
It should be `productId` or `productKey`.  It will be returned by the same API methods that return the localKey.

Reply:
Thanks, found it this time. I've updated this PR (and also rebased on top of current main)